src/indexPage/react-components/bot-landing/ChooseSection.jsx

```diff
@@ -0,0 +1,73 @@
+                        <div className='choose-wrapper-inner-description'>
+                            <h1>{translate('3 trade types')}</h1>
+                            <h2>{translate('Trade CFDs, digital options, and multipliers, a new, exciting trade type that boosts your potential profits with limited risk.')}</h2>
+                        </div>
+                    </div>
+                    <div className='choose-wrapper-inner-left-content'>
+                        <div className='choose-wrapper-inner-placeholder'>
+                            <img src="image/market_types.svg" />
+                        </div>
+                        <div className='choose-wrapper-inner-description'>
+                            <h1>{translate('6 market types')}</h1>
+                            <h2>{translate('Expand your portfolio with a wide range of markets such as forex, stocks, cryptocurrencies, synthetics, and more.')}</h2>
+                        </div>
+                    </div>
+                </div>
+                <div className='choose-wrapper-inner-right'>
+                    <div className='choose-wrapper-inner-right-content'>
+                        <div className='choose-wrapper-inner-placeholder'>
+                            <img src="image/fiat-on-ramp.svg" />
+                        </div>
+                        <div className='choose-wrapper-inner-description'>
+                            <h1 className='mb0'>{translate('Cryptocurrency deposits via fiat on-ramp')}</h1>
+                            <h2>{translate('Make crypto deposits easily via exchange services such as Changelly, Banxa, and XanPool.')}</h2>
+                        </div>
+                    </div>
+                    <div className='choose-wrapper-inner-right-content'>
+                        <div className='choose-wrapper-inner-placeholder'>
+                            <img src="image/peer-to-peer.svg" />
+                        </div>
+                        <div className='choose-wrapper-inner-description'>
+                            <h1>{translate('Peer-to-peer deposits and withdrawals')}</h1>
+                            <h2>{translate('Exchange your local currency with fellow traders to get funds in and out of your account with Deriv P2P.')}</h2>
+                        </div>
+                    </div>
+                    <div className='choose-wrapper-inner-right-content'>
+                        <div className='choose-wrapper-inner-placeholder'>
+                            <img src="image/deriv_academy.svg" />
+                        </div>
+                        <div className='choose-wrapper-inner-description'>
+                            <h1>{translate('Learn with Deriv Academy')}</h1>
+                            <h2>{translate('Enjoy complimentary articles and videos to help you learn the ropes of online trading.')}</h2>
+                        </div>
+                    </div>
+                </div>
```

Comment:
@rupato-deriv I would suggest mapping these 3 identical `choose-wrapper-inner-right-content` divs, and any other identical code blocks where it's applicable 🙂